src/Interface/Lists/Radarcord.ts

```diff
@@ -66,11 +66,70 @@ export default class RadarBotDirectory extends Service {
   getBotVotes(botID: IDResolvable, userID: IDResolvable) {
     return this._request(
       {
-        url: `/lastvoted/${Util.resolveID(userID)}/${Util.resolveID(botID)}`,
-        headers: { Authorization: this.token }
+        url: `/lastvoted/${Util.resolveID(userID)}/${Util.resolveID(botID)}`
       },
       {
-        requiresToken: true
+        requiresToken: false
+      }
+    )
+  }
+
+  /**
+   * Gets the username, ID and listed bots of user
+   * @param userID The user's ID
+   */
+  getUser(userID: IDResolvable) {
+    return this._request(
+      {
+        url: `/user/${Util.resolveID(userID)}`
+      },
+      {
+        requiresToken: false
+      }
+    )
+  }
+
+  /**
+   * Returns a 1 or 0 depending on whether the user has voted for the bot
```

Comment:
```suggestion
   * Gets whether the user has voted for the bot
```

Since this function doesn't actually return 0 or 1, it's best to stay generic, since the response body is already documented on each list's docs website